### DISCUSSION TITLE
Is it possible to override scheduler liveness probe arg?

### DISCUSSION THREAD

**Discussion by sfc-gh-cfraleigh** [airflow-helm/charts#595] [Questions & Answers]
We're having difficulty with the liveness probe for the scheduler, and would like to use a custom definition. Is it possible to override the helm chart version w/o specifying a new pod spec for all pods?

**Comment by thesuperzapper** [comment]
@cfraleig can you clarify what issues you are having with the scheduler liveness-prove so we can fix them for everyone?
